composer.json

```diff
@@ -30,15 +30,16 @@
         "ergebnis/composer-normalize": "^2.1",
         "guzzlehttp/psr7": "^1.0",
         "henrikbjorn/phpspec-code-coverage": "^1.0",
+        "laminas/laminas-diactoros": "^2.0",
+        "laminas/laminas-zendframework-bridge": "^1.0",
```

Comment:
why install the bridge for the tests? things would work fine without, right?

Reply:
Yes, we could drop the bridge here.